src/components/countriesList/CountriesList.js

```diff
@@ -0,0 +1,19 @@
+import { Link } from 'react-router-dom'
+
+import Countries from '../../countries.json'
+
+const CountriesList = ({cca3}) => {
+     
+    return (
+
+
+        <div className="col-5" Style="max-height: 90vh; overflow: scroll">
+            <div className="list-group">
+                {Countries.map(elm => <Link key={elm.cca3} className='list-group-item list-group-item-action' to={cca3}>{elm.flag} {elm.name.official}</Link> )} 
+            </div>
+        </div>
```

Comment:
```suggestion
        <div className="col-5" style={{ maxHeight: '90vh', overflow: 'scroll' }}>
            <div className="list-group">
                {Countries.map(elm => <Link to={`/countries/${elm.cca3}`} key={elm.cca3} className='list-group-item list-group-item-action' >{elm.flag} {elm.name.official}</Link>)}
            </div>
        </div>
```

Reply:
Con esto solucionarías los errores que me señalabas:

- A la prop to del link le estabas dando un valor incorrecto ya que cca3 existe solo dentro de elm.

- Para dar estilos inline, se usa la prop style (con la s en minúscula)